rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -3219,10 +3236,10 @@ def test_latest_upgrade_correctness(user_data_dir):
     this is just to reminds us not to forget to add create table statements.
     """
     msg_aggregator = MessagesAggregator()
-    base_database = 'v43_rotkehlchen.db'
+    base_database = 'v46_rotkehlchen.db'
```

Comment:
Can you also make it variable here? We follow a convention so you could still formulate the name with `ROTKEHLCHEN_DB_VERSION` and string manipulation right?